maglev.gemspec

```diff
@@ -26,14 +26,14 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_dependency 'globalid', '<= 0.4.1'
-  spec.add_dependency 'activemodel', '~> 5.2.0'
+  spec.add_dependency 'globalid', '<= 0.4.2'
```

Comment:
We might have to force this back to `0.4.1`. I remember 0.4.2 causing issues with Mongoidal

Reply:
Done.